src/common/ty/ty_printer.ml

```diff
@@ -223,12 +223,19 @@ let type_ ?(size=5000) ?(with_comments=true) t =
       | Get t -> fuse [
           Atom "get"; space;
           to_key key;
+          Atom "(";
```

Comment:
either add `softline` between the parens, or change to `wrap_and_indent (Atom "(", Atom ")") []`, to allow the line to wrap between the parens.